test/e2e/data/shared/addons/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
```

Comment:
Same consideration about introducing a dependency to a specific addon provider in CAPI E2E tests